bin/templates/cordova/lib/check_reqs.js

```diff
@@ -355,16 +355,16 @@ module.exports.check_android_target = function (originalError) {
 // Returns a promise.
 module.exports.run = function () {
     return Q.all([this.check_java(), this.check_android()]).then(function (values) {
-        console.log('ANDROID_HOME=' + process.env['ANDROID_HOME']);
```

Comment:
I don't think we should get rid of this log statement. This is information that can help a developer in case something goes wrong.

Reply:
Correct, the `check_android` check definitely checks this env var.

Reply:
I re-added the logging of ANDROID_HOME.